pulser-core/pulser/json/abstract_repr/deserializer.py

```diff
@@ -213,6 +213,7 @@ def _deserialize_operation(seq: Sequence, op: dict, vars: dict) -> None:
         seq.phase_shift_index(
             _deserialize_parameter(op["phi"], vars),
             *[_deserialize_parameter(t, vars) for t in op["targets"]],
+            basis=op["basis"],
```

Comment:
@HGSilveri What exactly were the implications of this bug for affected runs?

Reply:
Basically the basis was defaulting to `digital` every time. What happened generally was that an error was raised since there was no `Raman` declared channel